ui/src/lib.rs

```diff
@@ -12,51 +12,53 @@ mod view;
 
 enum Model {
     PageNotFound,
-    Title(page::title::Model),
+    Title,
```

Comment:
I deleted `Title`s state since it didnt really have any state.

Reply:
Nice